src/vs/workbench/contrib/replNotebook/browser/replEditorInput.ts

```diff
@@ -52,9 +57,17 @@ export class ReplEditorInput extends NotebookEditorInput implements ICompositeNo
 		@ICustomEditorLabelService customEditorLabelService: ICustomEditorLabelService,
 		@IInteractiveHistoryService public readonly historyService: IInteractiveHistoryService,
 		@ITextModelService private readonly _textModelService: ITextModelService,
-		@IConfigurationService configurationService: IConfigurationService
+		@IConfigurationService configurationService: IConfigurationService,
+		// --- Start Positron ---
+		@ICommandService commandService: ICommandService,
+		@IRuntimeSessionService runtimeSessionService: IRuntimeSessionService,
+		@ILogService logService: ILogService,
+		// --- End Positron ---
 	) {
-		super(resource, undefined, 'jupyter-notebook', {}, _notebookService, _notebookModelResolverService, _fileDialogService, labelService, fileService, filesConfigurationService, extensionService, editorService, textResourceConfigurationService, customEditorLabelService);
+		super(resource, undefined, 'jupyter-notebook', {}, _notebookService, _notebookModelResolverService, _fileDialogService, labelService, fileService, filesConfigurationService, extensionService, editorService, textResourceConfigurationService, customEditorLabelService,
+			// --- Start Positron ---
```

Comment:
I think the fence needs to be above `super`

Reply:
I was not sure at all on how to do this properly. Fence above the whole call seems nicer to me though!